librz/include/rz_util/rz_str.h

```diff
@@ -256,6 +256,18 @@ RZ_API const char *rz_str_sep(const char *base, const char *sep);
 RZ_API const char *rz_str_rsep(const char *base, const char *p, const char *sep);
 RZ_API char *rz_str_version(const char *program);
 
+typedef enum {
+	RZ_STR_STRINGIFY_DEFAULT = 0,
+	RZ_STR_STRINGIFY_ESCAPE_NL = 1 << 0,
+	RZ_STR_STRINGIFY_WRAP = 1 << 1,
+	RZ_STR_STRINGIFY_STOP_AT_NIL = 1 << 2,
+	RZ_STR_STRINGIFY_URLENCODE = 1 << 3,
+	RZ_STR_STRINGIFY_WIDE16_LE = 1 << 4,
+	RZ_STR_STRINGIFY_WIDE32_LE = 1 << 5,
```

Comment:
Add BE please

Reply:
i did not write the LE code, i kept the old one but is not complete and supports only ascii values which is wrong.